nvflare/private/fed/utils/fed_utils.py

```diff
@@ -236,3 +236,38 @@ def create_stats_pool_files_for_job(workspace: Workspace, job_id: str, prefix=No
         err = f"Failed to create stats pool summary file {summary_file}: {secure_format_exception(e)}"
     StatsPoolManager.close()
     return err
+
+
+def split_gpus(gpus) -> ([str], bool):
```

Comment:
Perhaps it's better to return a message instead if "bool" when the gpus string is invalid so help user know what's wrong.

Reply:
Changed.